systems/data-plan/api-gateway/pkg/rest/router.go

```diff
@@ -114,7 +115,7 @@ func formatDoc(summary string, description string) []fizz.OperationOption {
 }
 func (p *Router) getPackagesHandler(c *gin.Context) (*pb.GetByOrgPackageResponse, error) {
 	resp, err := p.clients.d.GetPackageByOrg(&pb.GetByOrgPackageRequest{
-		OrgId: TEMP_ORG_ID,
+		OrgID: strconv.Itoa(TEMP_ORG_ID),
```

Comment:
What is this temp ID for ? Again conv is from int to char.

Reply:
We were using it for testing purposes, but I have removed it and made it allowed in the path parameter instead.